docs/CODE_STYLE.md

```diff
@@ -1,47 +1,52 @@
   - Example: `var _example_e1`.
-- **Global** variables don't require any additional rules, as they already require a `global.` prefix to use.
+- **Global** variables require no additional prefix, as they already use `global.`.
   - Example: `global.example_e1`.
-- **Functions** don't require any prefixes, but to avoid accidental overlaps with instance variables:
-  - Name them with at least 2 words: `draw_something()` and not `drawing()`.
-  - And as an action: `draw_something()` and not `green_apple()`.
-  - Here we just try to follow default GM function naming scheme.
+- **Functions** don’t need prefixes but should:
+  - Use at least two words to avoid overlap with instance variables (e.g., `draw_something()`).
+  - Name functions as actions where possible (`draw_something()` vs. `green_apple()`).
+
 - **Macro** constants:
-  - A short `PREFIX_` should state their group in all caps.
-  - After the prefix, they should be written in snake_case.
-  - Examples: `COL_bright_red`, `COL_dark_green`; `DIR_left`, `DIR_up`.
-  - Here we just try to follow default GM macro naming scheme.
+  - To denote their group, use a short prefix in all caps (e.g., `PREFIX_`).
+  - After the prefix, use `snake_case`.
+  - Examples: `COL_bright_red`, `DIR_left`.
+
 - **Enum** constants:
-  - Name of the enum should start with an `e` prefix and be written in all caps.
-  - Name of the enum entries should use PascalCase.
-  - Example: `enum eCOLORS` with entries `DarkRed`, `Blue`, etc. 
+  - Enum names should start with an `e` prefix and be in all caps.
+  - Enum entries should use PascalCase.
+  - Example: `enum eCOLORS` with entries `DarkRed`, `Blue`, etc.
 
 ### General Styling:
-- It's recommended to initialize variables when declaring them. I.e `var variable = 0`, instead of just `var variable`.
-- It's also recommended to declaring each variable on a new line, avoiding `var variable1 = 0, variable2 = 2...`, to improve readability.
-- `&&` and `||` should be used instead of `and` and `or`.\
-This is to ensure compatibility with JavaScript formatters.
-- Indentation should be 4 **spaces**. Don't use tabs. Different editors interpret tabs differently.
-- Remember to use semicolons at the end of simple statements.
-- You can use [this](https://beautifier.io/) JavaScript formatter to format your code. The format it outputs fits our requirements. Just be sure to select:
+- Initialize variables when declaring them (`var variable = 0`).
+- Declare each variable on a new line, avoiding `var variable1 = 0, variable2 = 2...` for readability.
+- Use `&&` and `||` instead of `and` and `or` to ensure compatibility with JavaScript formatters.
+- Indentation should be 4 spaces (avoid tabs).
+- End simple statements with semicolons.
+- For string interpolation, use `$"text {variable}"` as it’s least error-prone and applies `string()` to `{variables}` automatically.
+- Use parentheses around conditions with multiple logical operators.
```

Comment:
I guess you copied the idea of:

> - Parenthesis in conditionals with logical operators should be used once around the whole condition, unless required to ensure the right order of operation.
>   - Right example: `if (something && something2 && something3)`.
>   - Wrong: `if (something) && (something2) && (something3)`.
>   - Wrong as well: `if ((something) && (something2) && (something3))`.

But in your variant it lost all meaning, imo.